plugins/tenv/tenv.plugin.zsh

```diff
@@ -0,0 +1,15 @@
+# Autocompletion for the FluxCD CLI (tenv).
```

Comment:
Why is it FluxCD?

Reply:
I wil lfix that .. I used the FLuxCD Plugin as base to make this one since they generate the autocomplete script the same way.

Reply:
Fixed .. thanks for the callout :)